### DISCUSSION TITLE
Help with Repository Transfer

### DISCUSSION THREAD

**Discussion by TorontoKayaks** [community/community#51286] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I made a request for my webdeveloper to transfer the code resource from his github to mine. When he sent the transfer the email from Github contains this message 

After accepting this repository transfer, you will lose access to:
  - Codeowners functionality.
  - Any existing wikis.
  - Pulse, Contributors, Community, Traffic, Commits, Code Frequency and Network on the Insights page.
  - Draft PRs.
  - Multiple assignees for issues and PRs.
  - Multiple PR reviewers.
  - Branch and tag protection rules.
  - Any existing Pages.
  
  Was this transfer completed correctly? Will I lose access to my website content/details? Should I accept this transfer request if my main intention is to gain control of my code resource so I can have ownership of my website and create the new changes I want.

**Comment by Anmol-Baranwal** [comment]
Yes, this transfer message is correct and normal for repository transfers on GitHub. When you accept the transfer, the current owner will lose access to the items listed in the message, but you will gain full ownership and control of the repository and its contents.

This means that you will have full control over the code resource and all of its details, including the ability to create new changes and updates. You will also have access to all of the repository's issues, pull requests, and other features.

However, it's important to note that you will lose access to the items listed in the transfer message, such as the existing wikis, branch and tag protection rules, and other insights and features. If any of these are important to you, you may want to consider creating a backup of them or transferring them separately.

Overall, accepting this transfer should allow you to gain full ownership and control of your website's code resource on GitHub, which is an important step in maintaining and updating your website.

**Reply by andrewbaxter** [reply]
> the current owner will lose access to the items listed in the message

The email is sent to the _new_ owner, not the current owner, and says

> you will lose access to:

which seems to imply that the _new owner_ won't have access to these things.  I think this is the source of the confusion (it confuses me too).

**Reply by hyliatechnologies** [reply]
> > the current owner will lose access to the items listed in the message
> 
> The email is sent to the _new_ owner, not the current owner, and says
> 
> > you will lose access to:
> 
> which seems to imply that the _new owner_ won't have access to these things. I think this is the source of the confusion (it confuses me too).

Hi. What happens then when you proceeded. I encounter the same confusion. Thanks for sharing with me

**Comment by BhathiyaPrasad** [comment]
When Repository transferring the previous owner get msg something like this and you will be having all access to the transferred repository
